news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md

```diff
@@ -0,0 +1,206 @@
+* [#300](https://github.com/PrestaShop/autoupgrade/pull/300): Improve report of errors and fix notices, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#302](https://github.com/PrestaShop/autoupgrade/pull/302): Display error when access is not granted, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#303](https://github.com/PrestaShop/autoupgrade/pull/303): Enable php-cs-fixer on the project, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#305](https://github.com/PrestaShop/autoupgrade/pull/305): Add Rollback explanation in readme, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#307](https://github.com/PrestaShop/autoupgrade/pull/307): Improve configuration getter for image backup, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+### Legal compliance
+
+* [#64](https://github.com/PrestaShop/ps_legalcompliance/pull/64): Unary operator spaces. Thank you [@MathiasReker](https://github.com/MathiasReker)
+
+
+### OnBoarding
+
+* [#56](https://github.com/PrestaShop/welcome/pull/56): Fix theme & logo, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#57](https://github.com/PrestaShop/welcome/pull/57): Update module version, README and LICENSE, by [@jolelievre](https://github.com/jolelievre)
+* [#58](https://github.com/PrestaShop/welcome/pull/58): Version updated to v6.0.0, by [@jolelievre](https://github.com/jolelievre)
+
+
+### Update of readme.md and license.md
+
+* [#13](https://github.com/PrestaShop/ps_banner/pull/13): Update Readme & License, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#14](https://github.com/PrestaShop/ps_banner/pull/14): Update Readme & License, by [@eternoendless](https://github.com/eternoendless)
+* [#23](https://github.com/PrestaShop/ps_checkpayment/pull/23): Update Readme & License, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#24](https://github.com/PrestaShop/ps_checkpayment/pull/24): Update Readme & License, by [@eternoendless](https://github.com/eternoendless)
+* [#42](https://github.com/PrestaShop/ps_wirepayment/pull/42): Update Readme & License, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#43](https://github.com/PrestaShop/ps_wirepayment/pull/43): Update Readme & License, by [@eternoendless](https://github.com/eternoendless)
+* [#27](https://github.com/PrestaShop/ps_categorytree/pull/27): Update Readme & License, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#28](https://github.com/PrestaShop/ps_categorytree/pull/28):  Update Readme & License, by [@eternoendless](https://github.com/eternoendless)
+* [#27](https://github.com/PrestaShop/productcomments/pull/27): Update Readme, by [@eternoendless](https://github.com/eternoendless)
+* [#28](https://github.com/PrestaShop/productcomments/pull/28): Add Readme and License, by [@eternoendless](https://github.com/eternoendless)
```

Comment:
We deal with different repositories here, what do you think about something like 

```suggestion
* [productcomments#28](https://github.com/PrestaShop/productcomments/pull/28): Add Readme and License, by [@eternoendless](https://github.com/eternoendless)
```

Reply:
good idea. that was to avoir to create a sub section for each module